setup.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from setuptools import setup, find_packages
+
+
+setup(
+    name='FoxPuppet',
+    version='0.1.0',
+    description='Foxpuppet for Firefox UI testing',
```

Comment:
I think we need a better description here. Something like 'Firefox user interface model for use with Selenium'. It's hard to describe, but I think we should mention Selenium and I like to avoid indicating that it's only for testing. I think there are many creative things users could do that aren't considered testing.